### DISCUSSION TITLE
Display relation manager table inside of a tab

### DISCUSSION THREAD

**Discussion by OvalMedia** [filamentphp/filament#4579] [Help] [answered]
Hello,

I built a relation similiar to the order/payment relation in the demo (https://demo.filamentphp.com/shop/orders/1/edit?activeRelationManager=0). In my case it's client/invoices.

Like in the demo the relations table is displayed on the bottom of the clients edit/view page. Is it possible to put this table inside of a tab instead?

Thank you

**Comment by danharrin** [comment, marked as answer]
This will be possible in Filament v3, when we move to use Livewire v3

**Reply by OvalMedia** [reply]
I assume there is no workaround for now?

**Reply by danharrin** [reply]
It's currently impossible, because you cant nest `<form>` elements in HTML

**Reply by Dontorpedo** [reply]
> This will be possible in Filament v3, when we move to use Livewire v3

will it be also possible to display widgets inside the tab? or can a custom field have such a functionality like a widget? (invoice editor or similar)
